### PR TITLE
Do not export hidden or non-selected objects

### LINKED ISSUE #17: Add "selection only" checkbox
Hi, awesome work on this glTF exporter so far!  Can I make a small feature request?

Please add a `Selection only` checkbox, similar to the STL exporter in Blender, such that only the selected objects are written to the exported file.  This would help for example when you need to create a file that just has a mesh or a model but no lights or cameras, etc.  Thanks!

### PR DESCRIPTION
**Pull request by onox**
Fixes #17.

### CODE REVIEW

**Comment by onox** [comment]
I can modify the PR to fix #17 if you want.

**Comment by Moguri** [comment]
There are use cases where you may want to export hidden nodes, so this should be controlled by a setting.

**Comment by onox** [comment]
I'll add a checkbox "Export Hidden Objects". Does that sound OK?

File: __init__.py [hunk 1]
```diff
@@ -59,6 +59,8 @@ class ExportGLTF(bpy.types.Operator, ExportHelper):
         check_extension = True
 
         #blendergltf settings
+        nodes_export_hidden = BoolProperty(name='Export Hidden Objects', default=False)
+        nodes_selected_only = BoolProperty(name='Selection Only', default=False)
```
Comment (Kupoman) [inline __init__.py:63]: These settings also need to be added to the default_settings dictionary near the top of blendergltf.py.

**Review by Kupoman — CHANGES_REQUESTED** [review]
I'm a little concerned about the performance of your implementation, but we can address that later if it becomes a problem. For now, I think just fixing the settings issue should be enough to get this into master.

**Review by Kupoman — APPROVED** [review]

**Pull request merged**